pipeline/terraform/main.tf

```diff
@@ -309,3 +309,69 @@ module "bastion" {
   subnet_id = "${module.vpc.public_subnets[0]}"
   vpc_id = "${module.vpc.vpc_id}"
 }
+
+// ~~~ RDS-Mysql ~~~
+
+data "aws_vpc" "default" {
+  default = true
+}
+
+data "aws_subnet_ids" "all" {
+  vpc_id = "${data.aws_vpc.default.id}"
+}
+
+data "aws_security_group" "default" {
+  vpc_id = "${data.aws_vpc.default.id}"
+  name   = "default"
+}
+
+module "rds" {
+  source  = "terraform-aws-modules/rds/aws"
+  version = "1.19.0"
+
+  # insert the 10 required variables here
+  identifier = "${var.identifier}"
+  engine            = "${var.engine}"
+  engine_version    = "${var.engine_version}"
+  instance_class    = "${var.instance_class}"
+  allocated_storage = "${var.allocated_storage}"
+  storage_encrypted = "${var.storage_encrypted}"
+  maintenance_window = "${var.maintenance_window}"
+  backup_window      = "${var.backup_window}"
+  username = "${var.username}"
+  password = "${var.password}"
+  port     = "${var.port}"
+  iam_database_authentication_enabled = true
+  vpc_security_group_ids = ["${aws_security_group.lb_sg.id}"]
+#["${data.aws_security_group.default.id}"]
+  backup_retention_period = 0
+
+  tags = {
+    Name       = "${local.resource_label}"
+    Environment = "${terraform.env}"
+  }
+
```

Comment:
This seems to result in a single instance (unless I'm mistaken). Our ECS cluster has two instances to demonstrate clustering capability. Does it make sense to do the same with RDS?

Reply:
Spoke with @jwilcoxen45. He agrees that we should show a robust implementation, infrastructure-wise (i.e., a cluster). What's it take to get there, and should it be multi-master?